src/directives/Onboard/CustomerQuote.tsx

```diff
@@ -30,7 +70,7 @@ function CustomerQuote({ hideQuote }: Props) {
                             : customerQuoteDark
                     }
                     width="85%"
-                    alt={intl.formatMessage({ id: 'company' })}
+                    alt={intl.formatMessage({ id: 'tenant.customer.quote' })}
```

Comment:
This should not have had the `alt` of `Estuary` so just typed up the quote.

Reply:
Previously, the image was solely the Estuary logo so the alternative text, I would argue, was appropriate.

Reply:
What do you mean? This file looks to have always been pointing to the customer quote.

![image](https://github.com/user-attachments/assets/d7861710-6c79-41ea-8ff3-4e4e24c64b88)